spec/csv_row_model/import_spec.rb

```diff
@@ -97,6 +116,23 @@ def self.name; "TwoLayerValid" end
               expect(instance.errors.full_messages).to eql ["Id can't be blank"]
             end
           end
+
+          context "with errors has a key with empty value" do
+            before do
+              expect(instance.csv_string_model).to receive(:valid?).at_least(1).times.and_wrap_original do |original, *args|
+                result = original.call(*args)
+                # this makes instance.csv_string_model.errors.messages = { id: [] }
+                instance.csv_string_model.errors[:id]
+                result
+              end
+            end
+
+            it "still shows the non-string validation" do
+              expect(subject).to eql false
+              expect(instance.csv_string_model.errors.messages).to eql(id: [])
+              expect(instance.errors.full_messages).to eql ["Id is too short (minimum is 5 characters)"]
+            end
+          end
```

Comment:
i forgot that this doesn't work. because of how errors propagate :P. adding it back in the next commit.